pkg/gardenlet/controller/backupbucket/reconciler_test.go

```diff
@@ -196,6 +197,7 @@ var _ = Describe("Controller", func() {
 	})
 
 	It("should not reconcile the extension BackupBucket if the secret data or extension spec hasn't changed", func() {
+		extensionBackupBucket.Status.LastOperation.LastUpdateTime = metav1.NewTime(time.Now().Add(time.Second * 10).UTC())
```

Comment:
```suggestion
		extensionBackupBucket.Status.LastOperation.LastUpdateTime = metav1.NewTime(time.Now().Add(time.Second * 1).UTC())
```

Isn't one second enough because that's the maximum rounding error we can have? Also please add a comment because no one will ever remember what happens here 😅

Reply:
One second should be enough, but the rounding off before comparing approach proposed by @shafeeqes and @plkokanov is maybe even better 👀